apps/condo/domains/user/schema/User.test.js

```diff
@@ -735,4 +738,58 @@ describe('Sensitive data search', () => {
             }, ['meta'])
         })
     })
+    test('Email address of service users can be read by another service users with direct access', async () => {
+        const admin = await makeLoggedInAdminClient()
+        const [rightsSet] = await createTestUserRightsSet(admin, {
+            canReadUsers: true,
+            canReadUserEmailField: true,
+        })
+        const firstServiceUser = await makeClientWithServiceUser({
+            rightsSet: { connect: { id: rightsSet.id } },
+        })
+        const secondServiceUser = await makeClientWithServiceUser()
+        const nonServiceUser = await makeClientWithNewRegisteredAndLoggedInUser()
+
+        const UserWithEmail = generateGQLTestUtils(generateGqlQueries('User', '{ id email deletedAt }'))
+
+        const readUser = await UserWithEmail.getOne(firstServiceUser, { id: secondServiceUser.user.id })
+        expect(readUser).toHaveProperty('id', secondServiceUser.user.id)
+        expect(readUser).toHaveProperty('email', normalizeEmail(secondServiceUser.userAttrs.email))
+        expect(readUser).toHaveProperty('deletedAt')
+        expect(readUser.deletedAt).toBeNull()
+
+        const accessDeniedCases = [
+            [firstServiceUser, nonServiceUser],
+            [secondServiceUser, firstServiceUser],
+            [secondServiceUser, nonServiceUser],
+            [nonServiceUser, secondServiceUser],
+        ]
+
+        for (const [client, target] of accessDeniedCases) {
+            await catchErrorFrom(async () => {
+                await UserWithEmail.getOne(client, { id: target.user.id })
+            }, (caught) => {
+                expect(caught).toMatchObject({
+                    name: 'TestClientResponseError',
+                    data: {
+                        objs: [
```

Comment:
why not use `expectToThrowAccessDeniedToFieldError`?

Reply:
Because it was not working :) And I was too lazy to fix it))